### DISCUSSION TITLE
%E8%B5%9E%E8%B5%8F%E8%AE%A1%E5%88%92/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#103] [Announcements]
# %E8%B5%9E%E8%B5%8F%E8%AE%A1%E5%88%92/

https://csmsapp.org/%E8%B5%9E%E8%B5%8F%E8%AE%A1%E5%88%92/

**Comment by RuijinShen** [comment]
请问如何赞赏？

**Reply by xichenpan** [reply]
感谢老板关注~因为项目一直在公益运行，当出现资金缺口时候会开启赞赏计划，本页面会出现赞赏二维码，达到目标金额后关闭
